### ISSUE TITLE
UnsatisfiedLinkError

### ISSUE THREAD

**Issue by amanthegreatone** [realm/realm-js#1559]
Hi,

I am getting the following error for one of my users. I think it happens on app load as I don't see my breadcrumbs for login in the bugsnag report.

```
java.lang.UnsatisfiedLinkError dlopen failed: cannot locate symbol "localeconv" referenced by "librealmreact.so"... 
    Runtime.java:330 java.lang.Runtime.load
    System.java:511 java.lang.System.load
    DirectorySoSource.java:71 com.facebook.soloader.DirectorySoSource.loadLibraryFrom
    DirectorySoSource.java:42 com.facebook.soloader.DirectorySoSource.loadLibrary
    SoLoader.java:299 com.facebook.soloader.SoLoader.loadLibraryBySoName
    SoLoader.java:247 com.facebook.soloader.SoLoader.loadLibrary
    RealmReactModule.java:43 io.realm.react.RealmReactModule.<clinit>
    RealmReactPackage.java:15 io.realm.react.RealmReactPackage.createNativeModules
    NativeModuleRegistryBuilder.java:106 com.facebook.react.NativeModuleRegistryBuilder.processPackage
    ReactInstanceManager.java:1191 com.facebook.react.ReactInstanceManager.processPackage
    ReactInstanceManager.java:1161 com.facebook.react.ReactInstanceManager.processPackages
    ReactInstanceManager.java:1094 com.facebook.react.ReactInstanceManager.createReactContext
    ReactInstanceManager.java:112 com.facebook.react.ReactInstanceManager.access$800
    ReactInstanceManager.java:922 com.facebook.react.ReactInstanceManager$4.run
    Thread.java:841 java.lang.Thread.run
```

realm: "^2.0.10"
react-native: "^0.50.3"
android 4.3 / device (HTC Desire 516 dual sim)

**Comment by amanthegreatone** [comment]
got a few more reports:

android 4.4.4 / Xiaomi 2014818
android 4.4.2 / vivo Y11

**Comment by kneth** [comment]
It has been fixed in 2.0.12.
